src/components/ContactCard/ContactGroups.jsx

```diff
@@ -8,17 +8,20 @@ import { withContactsMutations } from '../../connections/allContacts'
 export class ContactGroups extends React.Component {
   updateContactGroups = groups => {
     const modifiedContact = { ...this.props.contact }
-    modifiedContact.groups = groups.map(group => group._id)
+    modifiedContact.relationships.groups.data = groups
     this.props.updateContact(modifiedContact)
   }
 
   render() {
     const { allGroups, contact } = this.props
-    const contactGroups = contact.groups || []
+    const contactGroups =
```

Comment:
Maybe we could use [`_.get`](https://lodash.com/docs#get) here? It would make the code easier to read and maintain.